docs/content/quickstart/quick-start-docker.md

```diff
@@ -1,9 +1,19 @@
-<!-- vale off -->
-# Quickstart w. Docker
 
-This quick start is intended for developers, that have already configured their local development environment and familiarised them selves with all the technologies and frameworks used throughout the Medusa eco-system.
+# Quickstart using Docker
 
-If this is not the case, please head over to our Getting Started tutorial for a thorough walkthrough.
+In this document you will learn how to make a container of Medusa's app on Docker. Docker is an open source platform for building, deploying, and managing containerized applications.
+
+## Prerequisites
+
+### Node.js
+Medusa supports Node versions 14 and 16. You can check which version of Node you have by running the following command:
+```bash
```

Comment:
Can you add a new line before this? Same with a new line after the closing backticks